File: totosuki/ABC/B/259.py
import sys, math
input = sys.stdin.buffer.readline

a, b, d = map(int, input().split())

dist = math.sqrt(a**2 + b**2)
theta = math.atan2(b, a)

theta = theta + math.radians(d)

x = math.cos(theta) * dist
y = math.sin(theta) * dist

print(x, y)

File: OCH_Tao/ABC/B/259.js
const [a, b, d] = require("fs").readFileSync("/dev/stdin", "utf8").trim().split(" ").map(x => parseInt(x, 10));
const D = d * (Math.PI / 180);
console.log(a * Math.cos(D) - b * Math.sin(D), a * Math.sin(D) + b * Math.cos(D));